queries/highlights.scm

```diff
@@ -16,12 +16,11 @@
 
 [
   (implicit_type)
-  (nullable_type)
```

Comment:
I think this might have been a mistake in the query. I don't think we'd want to color `System.Int32?` differently than `System.Int32` in the below:
```C#
void M(System.Int32? b, System.Int32 c) { }
```